sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/MicrosoftTeamsAppIdentifier.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.common;
+
+import com.azure.core.util.CoreUtils;
+
+/**
+ * Communication identifier for Microsoft Teams Application
+ */
+public final class MicrosoftTeamsAppIdentifier extends CommunicationIdentifier {
+    private final String appId;
```

Comment:
```suggestion
    private final String teamsAppId;
```
it's approved name for JS and to maintain consistency across platforms , I think we should go with this name

Reply:
and we have change everywhere(f.e constructor params, setter, getters)

Reply:
Discussed offline.